about.html

```diff
@@ -0,0 +1,128 @@
+            every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
+        </div>
+        <div>
+          <p class="main-detail on-desktop about-detail">The Global Summit brings together the community of experts, academics, and activists who comprise the HopeGrower Gold affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the HopeGrower, free culture and open knowledge.
+            So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant HopeGrower together</p>
+        </div>
+        <div class="email-div">
+          <p class="email-contact">
+            Please contact us per Email for any further questions about HG Global Summit 2023!
+          </p>
+          <a href="mailto:tipu870@gmail.com" class="email">tipu870@gmail.com</a>
+        </div>
+      </div>
+      <div class="logo-section">
+        <h3 class="logo-heading">HopeGrower
+          Global Summit 2023 Logo
+        </h3>
+        <p class="logo-detail on-mobile">The logo of HG Global Summit 2023 was decided through the logo competition from 8, June
+          to 7, July.
+        </p>
+        <p class="logo-detail on-desktop">The logo of HG Global Summit 2023 was decided through the logo competition from 8, June to 7, July.
+          After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+          After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+        </p>
+        <div><img src="/img/logo.jpg" class="logo-img" alt="logo"></div>
+      </div>
+      <div class="past-summits">
+        <h3 class="past-heading">See the past HG Global Summits
+          <div class="underline"></div>
+        </h3>
+        <p class="past-detail">Take a look at the past two HG Global Summits which took place in Berlin and in Sydney
+        </p>
+        <div class="summit-img">
+          <div class="past-img"><img src="/img/conf-1.jpg" class="past-pic" alt="conf-1">
+            <div class="image-text">
+              <h3>2019</h3>
+              <p>HG Global Summit Berlin</p>
+            </div>
+          </div>
+          <div class="past-img"><img src="/img/conf-2.jpg" class="past-pic" alt="conf-1">
+            <div class="image-text">
+              <h3>2021</h3>
+              <p>HG Global Summit Sydney</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] you are missing the partner section of the mobile about page. please kindly follow the design
yours
![Screenshot (1080)](https://github.com/Mike111222/CapstoneProject/assets/65298048/eb6cd77f-c141-4498-babc-6783db2f3e45)
Behance design
![Screenshot (1079)](https://github.com/Mike111222/CapstoneProject/assets/65298048/e11b4678-d152-499a-ba48-6d136839a2fa)

Reply:
I have done the changes suggested by you sir. Thanks